src/Neo/SmartContract/Native/NeoToken.cs

```diff
@@ -283,9 +285,18 @@ private void SetGasPerBlock(ApplicationEngine engine, BigInteger gasPerBlock)
                 throw new ArgumentOutOfRangeException(nameof(gasPerBlock));
             if (!CheckCommittee(engine)) throw new InvalidOperationException();
 
-            uint index = engine.PersistingBlock.Index + 1;
-            StorageItem entry = engine.Snapshot.GetAndChange(CreateStorageKey(Prefix_GasPerBlock).AddBigEndian(index), () => new StorageItem(gasPerBlock));
+            var gp = new GasPerBlock() { Gas = gasPerBlock, Index = engine.PersistingBlock.Index + 1 };
```

Comment:
one argument is missing here, no?
The correct constructor requires 3

Reply:
`PreviousGas` default value is 0